habitat-lab/habitat/datasets/rearrange/rearrange_generator.py

```diff
@@ -886,8 +886,8 @@ def initialize_sim(self, scene_name: str, dataset_path: str) -> None:
             "rgb": {
                 "sensor_type": habitat_sim.SensorType.COLOR,
                 "resolution": camera_resolution,
-                "position": [0.0, 0.0, 0.0],
-                "orientation": [0.0, 0.0, 0.0],
+                "position": [0, 0, 0],
+                "orientation": [0, 0, 0.0],
```

Comment:
Nit: The Z component is still a float.

Reply:
These values were set to floats specifically [by this pr](https://github.com/facebookresearch/habitat-lab/commit/b97933e961c806c2aeb626c6c3ec0ff88dcc88ee)